tests/unittest/_torch/ray_orchestrator/single_gpu/test_llm_update_weights.py

```
@@ -142,119 +99,168 @@ def compare_logits(
+    ref_logits = hf_model.generate_batch_with_padding(
+        input_ids, attention_mask, position_ids, llm_responses, return_logits=True
+    )
     return llm_logits, ref_logits
 
 
+def process_and_copy_folder(src_folder, dst_folder, num_hidden_layers: int = 4):
+    if os.path.exists(dst_folder):
+        shutil.rmtree(dst_folder)
+    os.makedirs(dst_folder)
+
+    for root, dirs, files in os.walk(src_folder):
```

Ruff (0.14.10)

Loop control variable `dirs` not used within loop body

Rename unused `dirs` to `_dirs`

(B007)